components/cards/StateButton.tsx

```diff
@@ -0,0 +1,35 @@
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography/Typography'
+
+type StateButtonProps = {
+  state: 'active' | 'recruiting' | 'complete'
+}
+
+const StateButton = ({ state }: StateButtonProps) => {
+  const bgColor =
```

Comment:
nit: use some kind of mapping to help with readability, especially if we ever need to add yet another state.

Reply:
agreed! I addressed this in my newest commit :)